python/lsst/meas/astrom/createMatchMetadata.py

```diff
@@ -4,7 +4,25 @@
 from lsst.afw.geom import Box2D
 from lsst.afw.image.utils import getDistortedWcs
 
-__all__ = ["createMatchMetadata"]
+__all__ = ["MatchMetadata", "createMatchMetadata"]
+
+
+class MatchMetadata(PropertyList):
+    """Metadata required for unpersisting a match list"""
+    def __init__(self, ctrCoord, radius, filterName):
+        """!Ctor
+
+        @param[in] ctrCoord: Coordinates of center (lsst.afw.coord.IcrsCoord)
+        @param[in] radius: Minimum radius for selecting sources (lsst.afw.geom.Angle)
+        @param[in] filterName: Name of filter (str) or None
+        """
+        PropertyList.__init__(self)
+        ctrCoord = ctrCoord.toIcrs()
+        self.add('RA', ctrCoord.getRa().asDegrees(), 'field center in degrees')
+        self.add('DEC', ctrCoord.getDec().asDegrees(), 'field center in degrees')
+        self.add('RADIUS', radius.asDegrees(), 'field radius in degrees, minimum')
+        self.add('SMATCHV', 1, 'SourceMatchVector version number')
+        self.add('FILTER', filterName or "UNKNOWN", 'filter name for photometric data')
```

Comment:
I think you're referring to the line in astrometry.py:

        if filterName == "_unknown_":
            filterName = None

I have no idea where the `"_unknown_"` comes from, and I've never seen it in practise.  I think you added that code; do you know what it's for?